tests/web_digest_test.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+""" Python web digest test."""
+
+import unittest
+from unittest.mock import patch, MagicMock, Mock
+
+from agentscope.service import ServiceResponse
+from agentscope.service import web_load, webpage_digest
+from agentscope.service.service_status import ServiceExecStatus
+from agentscope.models import ModelWrapperBase, ModelResponse
+from agentscope.message import Msg
+
+
+class TestWebSearches(unittest.TestCase):
+    """ExampleTest for a unit test."""
+
+    @patch("requests.get")
+    def test_web_load(self, mock_get: MagicMock) -> None:
+        """test web_load function loading html"""
+        # Set up the mock response
+        mock_response = Mock()
+        mock_return_text = """
+            <!DOCTYPE html>
+            <html>
+            <body>
+                <div>
+                    <p>
+                    Some intro text about Foo. <a href="xxx">Examples</a>
```

Comment:
the <a> tag is not included in the expected result

Reply:
discussed in group chat.